
#######################################################################################################
                             PROJECT: matplotlib
#######################################################################################################
CLONING PROJECT:
✓ SUCCESSFUL! This project is already cloned: https://github.com/matplotlib/matplotlib
.......................................................................................................
BUGGY LINES:
Suspicious lines to bug1:
1511,1512,1513,1514,1532
Suspicious lines to bug2:
4396,4397,4479,4491,4492
Suspicious lines to bug3:
235
Suspicious lines to bug4:
1119,1133,1199,1215,2604
Suspicious lines to bug5:
4495
Suspicious lines to bug6:
4286
Suspicious lines to bug7:
1939
Suspicious lines to bug8:

Suspicious lines to bug9:

Suspicious lines to bug10:
827
Suspicious lines to bug11:
894,895,896,898,901
Suspicious lines to bug12:
1126,1129,1132,1135,1143
Suspicious lines to bug13:

Suspicious lines to bug14:
177
Suspicious lines to bug15:
72,32,1205,1206,1216
Suspicious lines to bug16:

Suspicious lines to bug17:

Suspicious lines to bug18:

Suspicious lines to bug19:

Suspicious lines to bug20:
257,258,259,260,1856
Suspicious lines to bug21:
3913,3937,3939,3943,3948
Suspicious lines to bug22:

Suspicious lines to bug23:
1510,1511
Suspicious lines to bug24:
1895
Suspicious lines to bug25:
1476,1482
Suspicious lines to bug26:
1909
Suspicious lines to bug27:
727
Suspicious lines to bug28:
3252
Suspicious lines to bug29:
1007,1008,1009,1010,1011
Suspicious lines to bug30:
380,381,382,384,385
.......................................................................................................

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+

                                          BUG: 1

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+
GIT RESET TO: c404d1f716e8aaefd4d7371ff49673e9c1f7f07c
bug_file = /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/backend_bases.py
bug_test_file = /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/tests/test_bbox_tight.py
TEST EXECUTION:
✓ SUCCESSFUL! TEST PASSES
✓ PySketchFix is started...
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

	                              PySketchFix®: Sketches are created.

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
PySketchFix will run with:
bug file: /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/backend_bases.py
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
The suspicious lines which are taken are: [1511, 1512, 1513, 1514, 1532]
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start transforming line: 1511
x Fail! Line: 1511 could not be transformed: '*renderer* by no-ops.  This is used by the tight-bbox-saving renderer,'
Start transforming line: 1512
x Fail! Line: 1512 could not be transformed: 'which needs to walk through the artist tree to compute the tight-bbox, but'
Start transforming line: 1513
x Fail! Line: 1513 could not be transformed: 'which the output file may be closed early.'
Start transforming line: 1514
x Fail! Line: 1514 could not be transformed: '"""'
Start transforming line: 1532
✓
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start creating 4 sketches.
✓
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
x Lines where no transformation is found:[1511, 1512, 1513, 1514]
✓ Number of created sketches:4
Time needed: 0.03 minutes.

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+

                                          BUG: 2

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+
GIT RESET TO: 2a3707d9c3472b1a010492322b6946388d4989ae
bug_file = /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/axes/_axes.py
bug_test_file = /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/tests/test_axes.py
TEST EXECUTION:
✓ SUCCESSFUL! TEST PASSES
✓ PySketchFix is started...
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

	                              PySketchFix®: Sketches are created.

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
PySketchFix will run with:
bug file: /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/axes/_axes.py
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
The suspicious lines which are taken are: [4396, 4397, 4479, 4491, 4492]
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start transforming line: 4396
x Fail! Line: 4396 could not be transformed: 'For non-filled markers, the *edgecolors* kwarg is ignored and'
Start transforming line: 4397
x Fail! Line: 4397 could not be transformed: 'forced to 'face' internally.'
Start transforming line: 4479
x Fail! Line: 4479 could not be transformed: 'edgecolors = 'face''
Start transforming line: 4491
✓
Start transforming line: 4492
✓
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start creating 3 sketches.
✓
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
x Lines where no transformation is found:[4396, 4397, 4479]
✓ Number of created sketches:3
Time needed: 0.04 minutes.

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+

                                          BUG: 3

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+
GIT RESET TO: 5e046f72ae82788788c7e9b9354b87b131891cd8
bug_file = /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/markers.py
bug_test_file = /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/tests/test_marker.py
TEST EXECUTION:
✓ SUCCESSFUL! TEST PASSES
✓ PySketchFix is started...
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

	                              PySketchFix®: Sketches are created.

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
PySketchFix will run with:
bug file: /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/markers.py
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
The suspicious lines which are taken are: [235]
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start transforming line: 235
✓
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start creating 4 sketches.
✓
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
x Lines where no transformation is found:[]
✓ Number of created sketches:4
Time needed: 0.0 minutes.

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+

                                          BUG: 4

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+
GIT RESET TO: 793c6b05381231371267b44b107726f3878e14f2
bug_file = /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/axes/_axes.py
bug_test_file = /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/tests/test_axes.py
TEST EXECUTION:
✓ SUCCESSFUL! TEST PASSES
✓ PySketchFix is started...
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

	                              PySketchFix®: Sketches are created.

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
PySketchFix will run with:
bug file: /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/axes/_axes.py
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
The suspicious lines which are taken are: [1119, 1133, 1199, 1215, 2604]
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start transforming line: 1119
x Fail! Line: 1119 could not be transformed: 'hlines(self, y, xmin, xmax, colors='k', linestyles='solid','
Start transforming line: 1133
x Fail! Line: 1133 could not be transformed: 'colors  list of colors, default 'k''
Start transforming line: 1199
x Fail! Line: 1199 could not be transformed: 'vlines(self, x, ymin, ymax, colors='k', linestyles='solid','
Start transforming line: 1215
x Fail! Line: 1215 could not be transformed: 'colors  list of colors, default 'k''
Start transforming line: 2604
x Fail! Line: 2604 could not be transformed: 'ecolor  color or list of color, default 'black''
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start creating 0 sketches.
✓
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
x Lines where no transformation is found:[1119, 1133, 1199, 1215, 2604]
✓ Number of created sketches:0
Time needed: 0.01 minutes.

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+

                                          BUG: 5

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+
GIT RESET TO: 49593b73854e10ace8f3c05343220328def6a328
bug_file = /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/axes/_axes.py
bug_test_file = /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/tests/test_axes.py
TEST EXECUTION:
✓ SUCCESSFUL! TEST PASSES
✓ PySketchFix is started...
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

	                              PySketchFix®: Sketches are created.

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
PySketchFix will run with:
bug file: /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/axes/_axes.py
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
The suspicious lines which are taken are: [4495]
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start transforming line: 4495
x Fail! Line: 4495 could not be transformed: 'linewidths = rcParams['lines.linewidth']'
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start creating 0 sketches.
✓
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
x Lines where no transformation is found:[4495]
✓ Number of created sketches:0
Time needed: 0.01 minutes.

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+

                                          BUG: 6

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+
GIT RESET TO: fb9e72309aeefb51d1a56ecbdb1e7399d105ff06
bug_file = /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/axes/_axes.py
bug_test_file = /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/tests/test_axes.py
TEST EXECUTION:
✓ SUCCESSFUL! TEST PASSES
✓ PySketchFix is started...
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

	                              PySketchFix®: Sketches are created.

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
PySketchFix will run with:
bug file: /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/axes/_axes.py
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
The suspicious lines which are taken are: [4286]
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start transforming line: 4286
✓
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start creating 18 sketches.
✓
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
x Lines where no transformation is found:[]
✓ Number of created sketches:18
Time needed: 0.01 minutes.

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+

                                          BUG: 7

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+
GIT RESET TO: 969513e2a5227331a2eb9e4bc4ba8448a0f9831d
bug_file = /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/colors.py
bug_test_file = /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/tests/test_colors.py
TEST EXECUTION:
✓ SUCCESSFUL! TEST PASSES
✓ PySketchFix is started...
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

	                              PySketchFix®: Sketches are created.

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
PySketchFix will run with:
bug file: /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/colors.py
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
The suspicious lines which are taken are: [1939]
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start transforming line: 1939
✓
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start creating 4 sketches.
✓
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
x Lines where no transformation is found:[]
✓ Number of created sketches:4
Time needed: 0.01 minutes.

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+

                                          BUG: 8

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+
x FAIL! Suspicious lines are empty!

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+

                                          BUG: 9

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+
x FAIL! Suspicious lines are empty!

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+

                                          BUG: 10

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+
GIT RESET TO: b31d64ce3910e8d297d8300690e459587f77181f
bug_file = /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/axis.py
bug_test_file = /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/tests/test_axes.py
TEST EXECUTION:
✓ SUCCESSFUL! TEST PASSES
✓ PySketchFix is started...
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

	                              PySketchFix®: Sketches are created.

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
PySketchFix will run with:
bug file: /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/axis.py
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
The suspicious lines which are taken are: [827]
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start transforming line: 827
x Fail! Line: 827 could not be transformed: '   '
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start creating 0 sketches.
✓
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
x Lines where no transformation is found:[827]
✓ Number of created sketches:0
Time needed: 0.01 minutes.

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+

                                          BUG: 11

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+
GIT RESET TO: f8459a513c3f67447ceb1a07c29760d504517ff2
bug_file = /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/text.py
bug_test_file = /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/tests/test_text.py
TEST EXECUTION:
✓ SUCCESSFUL! TEST PASSES
✓ PySketchFix is started...
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

	                              PySketchFix®: Sketches are created.

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
PySketchFix will run with:
bug file: /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/text.py
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
The suspicious lines which are taken are: [894, 895, 896, 898, 901]
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start transforming line: 894
✓
Start transforming line: 895
x Fail! Line: 895 could not be transformed: 'dpi_orig = self.figure.dpi'
Start transforming line: 896
✓
Start transforming line: 898
x Fail! Line: 898 could not be transformed: 'tx, ty = self._get_xy_display()'
Start transforming line: 901
✓
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start creating 728 sketches.
✓
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
x Lines where no transformation is found:[895, 898]
✓ Number of created sketches:728
Time needed: 0.06 minutes.

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+

                                          BUG: 12

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+
GIT RESET TO: e92685a26442bfced06067934f34c104487583a8
bug_file = /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/axes/_axes.py
bug_test_file = /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/tests/test_axes.py
TEST EXECUTION:
✓ SUCCESSFUL! TEST PASSES
✓ PySketchFix is started...
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

	                              PySketchFix®: Sketches are created.

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
PySketchFix will run with:
bug file: /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/axes/_axes.py
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
The suspicious lines which are taken are: [1126, 1129, 1132, 1135, 1143]
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start transforming line: 1126
✓
Start transforming line: 1129
✓
Start transforming line: 1132
x Fail! Line: 1132 could not be transformed: 'verts = [((thisxmin, thisy), (thisxmax, thisy))'
Start transforming line: 1135
x Fail! Line: 1135 could not be transformed: ' linestyles=linestyles, label=label)'
Start transforming line: 1143
x Fail! Line: 1143 could not be transformed: 'maxy = y.max()'
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start creating 27 sketches.
✓
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
x Lines where no transformation is found:[1132, 1135, 1143]
✓ Number of created sketches:27
Time needed: 0.03 minutes.

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+

                                          BUG: 13

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+
x FAIL! Suspicious lines are empty!

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+

                                          BUG: 14

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+
GIT RESET TO: dbc35a9d625e162445f864b9f463c9961888e901
bug_file = /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/text.py
bug_test_file = /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/tests/test_text.py
TEST EXECUTION:
✓ SUCCESSFUL! TEST PASSES
✓ PySketchFix is started...
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

	                              PySketchFix®: Sketches are created.

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
PySketchFix will run with:
bug file: /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/text.py
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
The suspicious lines which are taken are: [177]
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start transforming line: 177
x Fail! Line: 177 could not be transformed: '   '
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start creating 0 sketches.
✓
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
x Lines where no transformation is found:[177]
✓ Number of created sketches:0
Time needed: 0.01 minutes.

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+

                                          BUG: 15

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+
GIT RESET TO: 6a8e39f4eba28e99d7b1dd454b001b987dd0bbca
bug_file = /Users/daniel/Bachelorarbeit/Projects/matplotlib/examples/userdemo/colormap_normalizations.py
bug_test_file = /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/tests/test_colors.py
TEST EXECUTION:
✓ SUCCESSFUL! TEST PASSES
✓ PySketchFix is started...
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

	                              PySketchFix®: Sketches are created.

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
PySketchFix will run with:
bug file: /Users/daniel/Bachelorarbeit/Projects/matplotlib/examples/userdemo/colormap_normalizations.py
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
The suspicious lines which are taken are: [32, 72, 1205, 1206, 1216]
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start transforming line: 32
x Fail! Line: 32 could not be transformed: '   cmap='PuBu_r')'
Start transforming line: 72
x Fail! Line: 72 could not be transformed: '  vmin=-1.0, vmax=1.0),'
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start creating 0 sketches.
✓
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
x Lines where no transformation is found:[32, 72]
✓ Number of created sketches:0
Time needed: 0.01 minutes.

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+

                                          BUG: 16

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+
x FAIL! Suspicious lines are empty!

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+

                                          BUG: 17

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+
x FAIL! Suspicious lines are empty!

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+

                                          BUG: 18

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+
x FAIL! Suspicious lines are empty!

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+

                                          BUG: 19

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+
x FAIL! Suspicious lines are empty!

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+

                                          BUG: 20

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+
GIT RESET TO: 805f451b7122d1ef595c7a01592bb2f500aed41f
bug_file = /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/_constrained_layout.py
bug_test_file = /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/tests/test_axes.py
TEST EXECUTION:
✓ SUCCESSFUL! TEST PASSES
✓ PySketchFix is started...
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

	                              PySketchFix®: Sketches are created.

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
PySketchFix will run with:
bug file: /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/_constrained_layout.py
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
The suspicious lines which are taken are: [257, 258, 259, 260, 1856]
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start transforming line: 257
✓
Start transforming line: 258
✓
Start transforming line: 259
✓
Start transforming line: 260
✓
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start creating 719 sketches.
✓
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
x Lines where no transformation is found:[]
✓ Number of created sketches:719
Time needed: 0.06 minutes.

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+

                                          BUG: 21

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+
GIT RESET TO: e240493a899ac05cb992cdb88f5487386586090e
bug_file = /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/axes/_axes.py
bug_test_file = /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/tests/test_axes.py
TEST EXECUTION:
✓ SUCCESSFUL! TEST PASSES
✓ PySketchFix is started...
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

	                              PySketchFix®: Sketches are created.

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
PySketchFix will run with:
bug file: /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/axes/_axes.py
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
The suspicious lines which are taken are: [3913, 3937, 3939, 3943, 3948]
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start transforming line: 3913
✓
Start transforming line: 3937
✓
Start transforming line: 3939
x Fail! Line: 3939 could not be transformed: ''whiskerprops', whiskerprops)'
Start transforming line: 3943
x Fail! Line: 3943 could not be transformed: ''flierprops', flierprops)'
Start transforming line: 3948
x Fail! Line: 3948 could not be transformed: 'removed_prop = 'marker' meanline 'linestyle''
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start creating 9 sketches.
✓
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
x Lines where no transformation is found:[3939, 3943, 3948]
✓ Number of created sketches:9
Time needed: 0.03 minutes.

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+

                                          BUG: 22

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+
x FAIL! Suspicious lines are empty!

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+

                                          BUG: 23

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+
GIT RESET TO: bb6a4af984778dbac55c8391ae29fa2b5b201361
bug_file = /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/axes/_base.py
bug_test_file = /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/tests/test_axes.py
TEST EXECUTION:
✓ SUCCESSFUL! TEST PASSES
✓ PySketchFix is started...
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

	                              PySketchFix®: Sketches are created.

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
PySketchFix will run with:
bug file: /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/axes/_base.py
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
The suspicious lines which are taken are: [1510, 1511]
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start transforming line: 1510
✓
Start transforming line: 1511
✓
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start creating 41 sketches.
✓
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
x Lines where no transformation is found:[]
✓ Number of created sketches:41
Time needed: 0.03 minutes.

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+

                                          BUG: 24

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+
GIT RESET TO: 9a5473dbac05f3d6773b42c8e16d58a7dc3159b8
bug_file = /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/axis.py
bug_test_file = /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/tests/test_axes.py
TEST EXECUTION:
✓ SUCCESSFUL! TEST PASSES
✓ PySketchFix is started...
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

	                              PySketchFix®: Sketches are created.

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
PySketchFix will run with:
bug file: /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/axis.py
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
The suspicious lines which are taken are: [1895]
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start transforming line: 1895
x Fail! Line: 1895 could not be transformed: 'setter(self, max(vmin, vmax, oldmax), min(vmin, vmax, oldmin),'
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start creating 0 sketches.
✓
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
x Lines where no transformation is found:[1895]
✓ Number of created sketches:0
Time needed: 0.0 minutes.

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+

                                          BUG: 25

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+
GIT RESET TO: 9a5473dbac05f3d6773b42c8e16d58a7dc3159b8
bug_file = /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/collections.py
bug_test_file = /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/tests/test_collections.py
TEST EXECUTION:
✓ SUCCESSFUL! TEST PASSES
✓ PySketchFix is started...
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

	                              PySketchFix®: Sketches are created.

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
PySketchFix will run with:
bug file: /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/collections.py
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
The suspicious lines which are taken are: [1476, 1482]
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start transforming line: 1476
✓
Start transforming line: 1482
x Fail! Line: 1482 could not be transformed: '  orientation.lower() == 'horizontal')'
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start creating 29 sketches.
✓
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
x Lines where no transformation is found:[1482]
✓ Number of created sketches:29
Time needed: 0.02 minutes.

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+

                                          BUG: 26

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+
GIT RESET TO: 04d9d28b820af0b4df230a3c67314ed5b0af8fdd
bug_file = /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/axis.py
bug_test_file = /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/tests/test_axes.py
TEST EXECUTION:
✓ SUCCESSFUL! TEST PASSES
✓ PySketchFix is started...
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

	                              PySketchFix®: Sketches are created.

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
PySketchFix will run with:
bug file: /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/axis.py
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
The suspicious lines which are taken are: [1909]
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start transforming line: 1909
x Fail! Line: 1909 could not be transformed: 'setter(self, max(vmin, vmax, oldmax), min(vmin, vmax, oldmin),'
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start creating 0 sketches.
✓
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
x Lines where no transformation is found:[1909]
✓ Number of created sketches:0
Time needed: 0.0 minutes.

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+

                                          BUG: 27

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+
GIT RESET TO: 11269123d516cda369764a081ddfb8c1a10ddc53
bug_file = /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/colorbar.py
bug_test_file = /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/tests/test_colorbar.py
TEST EXECUTION:
✓ SUCCESSFUL! TEST PASSES
✓ PySketchFix is started...
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

	                              PySketchFix®: Sketches are created.

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
PySketchFix will run with:
bug file: /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/colorbar.py
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
The suspicious lines which are taken are: [727]
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start transforming line: 727
✓
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start creating 3 sketches.
✓
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
x Lines where no transformation is found:[]
✓ Number of created sketches:3
Time needed: 0.01 minutes.

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+

                                          BUG: 28

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+
GIT RESET TO: 896fb8140a600341ebb5eaa4191584f23df2d2a0
bug_file = /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/axes/_base.py
bug_test_file = /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/tests/test_axes.py
TEST EXECUTION:
✓ SUCCESSFUL! TEST PASSES
✓ PySketchFix is started...
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

	                              PySketchFix®: Sketches are created.

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
PySketchFix will run with:
bug file: /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/axes/_base.py
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
The suspicious lines which are taken are: [3252]
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start transforming line: 3252
✓
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start creating 2 sketches.
✓
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
x Lines where no transformation is found:[]
✓ Number of created sketches:2
Time needed: 0.01 minutes.

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+

                                          BUG: 29

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+
GIT RESET TO: cdf9e30e4f3fb7747b178ee9c3849dfdebee7bd0
bug_file = /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/axis.py
bug_test_file = /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/tests/test_axes.py
TEST EXECUTION:
✓ SUCCESSFUL! TEST PASSES
✓ PySketchFix is started...
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

	                              PySketchFix®: Sketches are created.

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
PySketchFix will run with:
bug file: /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/axis.py
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
The suspicious lines which are taken are: [1007, 1008, 1009, 1010, 1011]
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start transforming line: 1007
x Fail! Line: 1007 could not be transformed: 'a, b = self.get_view_interval()'
Start transforming line: 1008
✓
Start transforming line: 1009
✓
Start transforming line: 1010
x Fail! Line: 1010 could not be transformed: ''
Start transforming line: 1011
✓
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start creating 26 sketches.
✓
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
x Lines where no transformation is found:[1007, 1010]
✓ Number of created sketches:26
Time needed: 0.06 minutes.

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+

                                          BUG: 30

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+
GIT RESET TO: 3b26ee6f6b31bc0cca4be0407dbb40f44756030d
bug_file = /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/colors.py
bug_test_file = /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/tests/test_colors.py
TEST EXECUTION:
✓ SUCCESSFUL! TEST PASSES
✓ PySketchFix is started...
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

	                              PySketchFix®: Sketches are created.

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
PySketchFix will run with:
bug file: /Users/daniel/Bachelorarbeit/Projects/matplotlib/lib/matplotlib/colors.py
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
The suspicious lines which are taken are: [380, 381, 382, 384, 385]
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start transforming line: 380
✓
Start transforming line: 381
✓
Start transforming line: 382
✓
Start transforming line: 384
✓
Start transforming line: 385
x Fail! Line: 385 could not be transformed: 'lut = np.concatenate(['
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start creating 89 sketches.
✓
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
x Lines where no transformation is found:[385]
✓ Number of created sketches:89
Time needed: 0.05 minutes.
#######################################################################################################
Results with bug numbers:
Project:matplotlib
Failing:[]
Passing:[1, 2, 3, 4, 5, 6, 7, 10, 11, 12, 14, 15, 20, 21, 23, 24, 25, 26, 27, 28, 29, 30]
Empty Suspicious Lines:[8, 9, 13, 16, 17, 18, 19, 22]
No bug information is found:[]
